### ISSUE TITLE
Some doubts regarding beam size & beam alpha, training parameters in public_param.py and those reported in paper

### ISSUE THREAD

**Issue by agenius5** [google-research/pegasus#129]
I used following code for evaluation on fine-tuned checkpoints available on google cloud for aeslc dataset.

`!python3 pegasus/pegasus/bin/evaluate.py \
--params=aeslc_transformer \
--param_overrides=vocab_filename='/content/drive/My Drive/ckpt/c4.unigram.newline.10pct.96000.model',batch_size=16, beam_size=8,beam_alpha=0.6 \
--model_dir='/content/ckpt/aeslc' \
--evaluate_test=True \
--full=True`

I kept batch size as 16, beam_size as 8 and beeam_aplha as 0.6 acc to paper. -param_override was not working so I had changed the values in public_param.py and got following results. Evaluation time also increased.

`R1-F, R2-F, RL-F = 0.375311/0.212271/0.363064`

Very close but still a little off. Why? Now I have following questions:

1. What batch size did you use for evaluation and does it affect evaluation result?

2. Does `beam_size` and `beam_aplha` are also used during training or only during evaluation while generating summaries?

3. Why you guys kept `learning rates`, `train_steps` and `batch size` different in dataset resgistries in `public_param.py` than those reported in paper? On what basis these new `train_steps`, `learning rate` and `batch size` are given in `public_param.py` for all the datasets

Also any other parameter are changed too in `contrib_training.HParam(...)` parameter list?

**Comment by JingqingZ** [comment]
Please refer to our paper Appendix C for a complete (and final) list of hyper-parameters we used.

Some grid-search experiments were run with `--param_overrides` thus the final hyper-parameters we reported in paper can be different from those in code.

**Comment by agenius5** [comment]
Okay I think I finally understand things. So basically all the numbers reported are obtained with the parameters listed in Appendix C. Now pleaset tell me just one more things:

In low resource setting, model was trained till 2000 steps but test set was evaluated on that checkpoint which gave best validation set numbers. So all low resource numbers reported are not necessarily obtained by evaluating test set on exact 2000 step mark. Right?

**Comment by JingqingZ** [comment]
> In low resource setting, model was trained till 2000 steps but test set was evaluated on that checkpoint which gave best validation set numbers. So all low resource numbers reported are not necessarily obtained by evaluating test set on exact 2000 step mark. Right?


That is correct.

**Comment by agenius5** [comment]
@JingqingZ Nice! Thanks :)
